locales/od_IN/LC_MESSAGES/duckduckgo.po

```diff
@@ -575,20 +576,26 @@ msgid ""
 "Browse as usual, and we'll take care of the rest. We bundled our search "
 "engine, tracker blocker, and encryption enforcer into one %s%s Extension%s."
 msgstr ""
+"ନିୟମିତ ଭଳି ବ୍ରାଉଜ କରନ୍ତୁ, ଆମେ ବାକି ସବୁ ଦେଖିନବୁ । ଆମେ ଆମର ସନ୍ଧାନ ଇଞ୍ଜିନ, "
+"ଅନୁସରଣକାରୀ ଅବରୋଧକାରୀ ଏବଂ ଏନ୍କ୍ରିପସନ ଏନ୍ଫୋର୍ସର କୁ ଏକାଠି କରି ଗୋଟିଏ  %s%s "
+"ଏକଷ୍ଟେନ୍ସନରେ %s ରଖିଛୁ ।"
 
 msgid ""
-"Browse as usual, and we’ll take care of the rest. Get bundled private "
-"search, tracker blocking, and site encryption, all in one download, for "
-"%smajor browsers%s."
+"Browse as usual, and we’ll take care of the rest. Get bundled private search,"
+" tracker blocking, and site encryption, all in one download, for %smajor "
+"browsers%s."
 msgstr ""
+"ନିୟମିତ ଭଳି ବ୍ରାଉଜ କରନ୍ତୁ, ଆମେ ବାକି ସବୁ ଦେଖିନବୁ । ଆମେ ଆମର ସନ୍ଧାନ ଇଞ୍ଜିନ, "
+"ଅନୁସରଣକାରୀ ଅବରୋଧକାରୀ ଏବଂ ଏନ୍କ୍ରିପସନ ଏନ୍ଫୋର୍ସର କୁ ଏକାଠି କରି ଗୋଟିଏ  %s%s "
+"ସ୍ମାର୍ଟ ବ୍ରାଉଜରରେ%s ରଖିଛୁ ।"
```

Comment:
Would you mind checking this again? It looks like some formatting got lost. In the original string, `%smajor browsers%s` is enclosed in `%s` tags. This must be kept in the translation. Thanks!